### PR TITLE
fix(mme): Extended pr7912 that added lbi to OAI CUPS path

### PR DESCRIPTION
**Pull request by ulaskozat**
## Summary

PR #7912 adds support for LBI on GTPv2 based S11 path. It had some build issues and additional review comments. To get unblocked, raising a new PR to fix these.

## Test Plan

Integ tests and OAI CI pipeline.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/c/core/oai/tasks/nas/esm/sap/esm_recv.c [hunk 1]
```diff
@@ -523,13 +523,14 @@ esm_cause_t esm_recv_pdn_disconnect_request(
     OAILOG_FUNC_RETURN(LOG_NAS_ESM, ESM_CAUSE_INVALID_EPS_BEARER_IDENTITY);
   }
 
+  bool no_delete_tunnel = (ue_mm_context_p->nb_active_pdn_contexts > 1);
```
Comment (ulaskozat) [inline lte/gateway/c/core/oai/tasks/nas/esm/sap/esm_recv.c]: this is always true due to line 470. But, I think we should set this always to false as we should not delete the gtpv2c tunnel until all sessions got deleted.

File: lte/gateway/c/core/oai/tasks/mme_app/mme_app_detach.c [hunk 2]
```diff
@@ -201,11 +202,20 @@ void mme_app_handle_detach_req(const mme_ue_s1ap_id_t ue_id) {
...
+        cid_delete = i;
+      }
+    }
     for (pdn_cid_t i = 0; i < MAX_APN_PER_UE; i++) {
       if (ue_context_p->pdn_contexts[i]) {
         // Send a DELETE_SESSION_REQUEST message to the SGW
         mme_app_send_delete_session_request(
-            ue_context_p, ue_context_p->pdn_contexts[i]->default_ebi, i);
+            ue_context_p, ue_context_p->pdn_contexts[i]->default_ebi, i,
+            (cid_delete != i));
```
Comment (ardzoht) [inline lte/gateway/c/core/oai/tasks/mme_app/mme_app_detach.c]: we should update `no_delete` bool and use it here?

Comment (ulaskozat) [reply lte/gateway/c/core/oai/tasks/mme_app/mme_app_detach.c]: yeah, was just updating those. I am changing couple of more things. I will ping you :)

**Review by ardzoht — APPROVED** [review]
lgtm

**Pull request merged**